content/code_of_conduct.md

```diff
@@ -45,6 +45,6 @@ As members of the community,
 
 This code of conduct has been adapted from the _Astropy Code of Conduct_, which in turn uses parts of the _Python Software Foundation (PSF)_ code of conduct.
 
-**To report any violations of the code of conduct, please contact a member of the [TARDIS core team](../team/community_roles) (the email tardis.supernova.code@gmail.com is monitored by the core team) or the Ombudsperson (see the [team page](../team/community_roles); who is outside of the TARDIS collaboration and will treat reports confidentially).**
+**To report any violations of the code of conduct, please contact a member of the [TARDIS core team](./people/core.md) (the email tardis.supernova.code@gmail.com is monitored by the core team) or the Ombudsperson (see the [team page](./people/core.md#ombudsperson--anchor-offset); who is outside of the TARDIS collaboration and will treat reports confidentially).**
```

Comment:
adding .md doesn't work on my osx system but these urls work 
(../people/core)
(../people/core#ombudsperson)

Reply:
I modified in commits:
- [779be33](https://github.com/tardis-sn/tardis-sn.github.io/pull/80/commits/779be3368987d8761ab2fc7818ee3cfe1749b59c)
- [584a0ef](https://github.com/tardis-sn/tardis-sn.github.io/pull/80/commits/584a0ef42ca442a77591da49b7f599b58621fb0a)